app/filters/Filters.scala

```diff
@@ -1,14 +1,15 @@
 package filters
 
 import akka.stream.Materializer
+import com.kelkoo.play.filters.AccessLogFilter
```

Comment:
Revert this change, as the accessLogFilter is Kelkoo specific and not opensourced